src/main/java/parser/DateTimeExtractor.java

```diff
@@ -0,0 +1,55 @@
+package parser;
+
+import java.text.ParseException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This DateTimeExtractor class allows the identification of dates and is used
+ * to print out dates and times.
+ *
+ * @author Sai Ganesh Suresh
+ * @version v2.0
+ */
+public class DateTimeExtractor {
+
+    public static DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+    private static LocalDateTime dateEvent;
+    private static LocalDateTime dateDeadline;
+    private static LocalDateTime datePostpone;
```

Comment:
If you want to only assign and return, what is the purpose of having this many variables of same type. If you have any future use, then it should be fine.